.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: CI Release
+on:
+  release:
+    types: [created]
+jobs: 
+  test_release: 
+    name: Check release
+    runs-on: ubuntu-18.04
+    steps: 
+      - uses: actions/checkout@master
+      - name: Remove package-lock
+        run: rm -rf package-lock.json
+      - name: Install dependence
+        run: npm install
+      - name: build 
+        if: success()
+        run: npm run build
+      - name: Check version
+        if: success()
+        run: npm run check:release
+      - name: Process package
+        if: success()
+        run: npm pack 
+      - name: Upload binaries to release
+        uses: svenstaro/upload-release-action@v1-release
+        with: 
+          repo_token: ${{ secrets.ACCESS_TOKEN }}
+          file: kintone-kintone-ui-component-0.0.30.tgz
```

Comment:
@Pham-Gia-Huong 
make kintone-kintone-ui-component-0.0.30.tgz name dynamic and not hard-coded